Formula/mp.rb

```diff
@@ -0,0 +1,13 @@
+class Mp < Formula
+  desc "Minimum Profit editor by Angel Ortega"
+  homepage "https://triptico.com/software/mp.html"
+  url "https://github.com/juiceghost/homebrew-mp/archive/v3.2.13.tar.gz"
+  sha256 "176aea01f4334605ea0cc89caf7ab0a0c600c8367b8a4779fcb7d54983af1dac"
+  def install
+    system "make", "mp"
+    bin.install "mp"
+  end
+  test do
+    assert_match "3.2.13", shell_output("#{bin}/mp --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
`Usage: mp [options] [file [file ...]]

Options:

 -t|--tag [tag] 	Edits the file where tag is defined
 -w|--word-wrap [col]	Sets wordwrapping in column col
 -ts|--tab-size [size]	Sets tab size
 --mouse		Activate mouse usage for cursor positioning
 --col80		Marks column # 80
 -hw|--hardware-cursor	Activates the use of hardware cursor
 --version		Shows version
 -h|--help		This help screen
 -bw|--monochrome	Monochrome
 -ai|--autoindent	Sets automatic indentation mode
 -nt|--no-transparent	Disable transparent mode (eterm, aterm, etc.)
 -l|--lang [lang]	Language selection
 -m|--mode [mode]	Syntax-hilight mode

--lang: en es de it nl
--mode: plain c perl sh ruby php sql ps python html xml po mail man ebuild grub
`

These are the options it takes, am open to suggestions on how to test it further, but being an interactive console app i think version is as good as it gets. For comparison, this is the test of htop:
`    pipe_output("#{bin}/htop", "q", 0)`